addOns/postman/src/main/java/org/zaproxy/addon/postman/ImportDialog.java

```diff
@@ -83,10 +84,10 @@ public ImportDialog(JFrame parent) {
                 LayoutHelper.getGBC(2, fieldsRow, 1, 0.5, new Insets(0, 4, 4, 0)));
         fieldsRow++;
         fieldsPanel.add(
-                new JLabel(Constant.messages.getString("postman.importDialog.labelTarget")),
+                new JLabel(Constant.messages.getString("postman.importDialog.labelVariables")),
                 LayoutHelper.getGBC(0, fieldsRow, 1, 0.5, new Insets(4, 0, 4, 4)));
         fieldsPanel.add(
-                getTargetField(),
+                getVariablesField(),
```

Comment:
Should be adding a new field not replace the target? (Even if the target is not used right now it should at some point, maybe hide instead of replacing).